cookbooks/openssh/attributes/default.rb

```diff
@@ -1 +1,2 @@
 default[:openssh][:port] = 22
+default[:openssh][:disable_password_authentication] = true
```

Comment:
I would reverse the sense of this and just call it `:password_authentication` which is both short and matches the actual ssh configuration option.